core/src/main/java/org/opensearch/sql/planner/physical/catalog/CatalogTableScan.java

```diff
@@ -51,7 +51,7 @@ public void open() {
     for (Catalog catalog : catalogs) {
```

Comment:
will these be renamed or in code they will keep as catalogs?

Reply:
yes...will raise PR sometime this week.